module5/TaskListApp/TaskListApp/App/TaskListAppApp.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TaskListAppApp.swift
+//  TaskListApp
+//
+//  Created by Peter Rice on 12/22/23.
+//
+
+import SwiftUI
+
+@main
+struct TaskListAppApp: App {
+  @StateObject var tasksStore = TasksStore()
```

Comment:
Your app has one main screen that has links to all child screens. Consider putting the `@StateObject` initializer into the `TaskListView` screen instead of up here.

Reply:
OK fair point.